matflow/param_classes/load.py

```diff
@@ -167,6 +173,10 @@ def _validate(self):
         if isinstance(self.target_vel_grad, list):
             self.target_vel_grad = masked_array_from_list(self.target_vel_grad)
 
+        if self.rotation is not None:
+            if isinstance(self.rotation, list):
+                self.rotation = masked_array_from_list(self.rotation)
+
```

Comment:
Since this doesn't need to be parsed as a masked array, I think this can just be:
```suggestion
        if self.rotation is not None:
            self.rotation = np.asarray(self.rotation)
```